docs/content/release_notes/csi_rn_whatsnew.md

```diff
@@ -1,22 +1,13 @@
-# What's new in 1.7.0
+# What's new in 1.8.0
 
-IBM® block storage CSI driver 1.7.0 introduces the enhancements that are detailed in the following section.
+IBM® block storage CSI driver 1.8.0 introduces the enhancements that are detailed in the following section.
 
-**General availability date:** 30 September 2021
+**General availability date:** 30 December 2021
```

Comment:
please verify this date with Eran

Reply:
the final date will be changed later and in a later PR. thanks